src/scss/components/_inputs.scss

```
@@ -136,6 +141,14 @@ $input-select-margin-right: 1;
   }
 }
 
+.usa-radio__input--tile:disabled + .usa-radio__label--illustrated,
+.usa-checkbox__input--tile:disabled + .usa-checkbox__label--illustrated {
+  .usa-radio__image,
+  .usa-checkbox__image {
+    filter: invert(45%) sepia(1%) saturate(0%) hue-rotate(232deg) brightness(100%) contrast(80%);
+  }
+}
```

I might suggest avoiding the nesting here, so that (1) the image styles only apply for the correct combination of radio + radio or checkbox + checkbox, and (2) to reduce the size of the output ([see comparison](https://www.sassmeister.com/gist/82331cc1f75fa50d8f7f7938bfee1ad9)).

```suggestion
.usa-radio__input--tile:disabled + .usa-radio__label--illustrated .usa-radio__image,
.usa-checkbox__input--tile:disabled + .usa-checkbox__label--illustrated .usa-checkbox__image {
  filter: invert(45%) sepia(1%) saturate(0%) hue-rotate(232deg) brightness(100%) contrast(80%);
}
```